tests/pingpong-shared/pingpong.edl

```diff
@@ -1,18 +1,20 @@
+
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
 enclave {
     trusted {
         public void Ping(
             [in, string] const char* in,
-            [in, out, string] char* out);
-
+            [in, out, string] char* out,
+			int out_length);
     };
 
     untrusted {
         void Pong(
             [in, string] const char* in,
-            [in, out, string] char* out);
+            [in, out, string] char* out,
+			int out_length);
```

Comment:
Nit: use same indentation as the rest of the file (spaces, not tabs)

Reply:
Ah I see! No wonder the format-code does not pick out the problem. I'll change it as well as the settings in my editor.